aws-code-build/run_local/run_local.env

```diff
@@ -0,0 +1,5 @@
+FLAVOR=python-3.8-minimal-EXASOL-6.2.0
```

Comment:
is EXASOL 6.2.0 still correct?

Reply:
More or less, we already had a discussion with PM and as a consequence opened https://github.com/exasol/script-languages-release/issues/566 to fix all custom flavor names.